### DISCUSSION TITLE
Use AZCopy to drop the log files into a storage account

### DISCUSSION THREAD

**Discussion by jfieldUA** [PSAppDeployToolkit/PSAppDeployToolkit#1097] [Q&A] [answered]
One thing I'd like to see is a way to gather the log files for failed deployments in a central place. I've used AZCopy for that in the past and it worked well.

**Comment by mjr4077au** [comment, marked as answer]
This isn't something that we'd likely carry in the main project, but it'd be something perfect for a PSAppDeployToolkit extension module.
